smf/clickhouse_server/method_script.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
```

Comment:
I know this file was mostly just moved over, but I don't expect it to end up looking like this at all in the very near future. It will only start `clickhouse-admin` which will then wait for a `ReplicaConfig`. If a `clickhouse-config.xml` already exists then it should start clickhouse with that config. `clickhouse-admin` will continue waiting for updates indefinitely and rewrite `clickhouse-config.xml` when the configuration changes. Clickhouse will automatically reload this configuration.

Reply:
ah yeah, definitely! I expect this to be gone soon.